### PR TITLE
Update media search filter API docs

### PR DESCRIPTION
**Pull request by NolanVH**
### Description of Changes
This PR adds a small addition to the API docs describing how to use the media search filter.

### Checklist
- ~README / documentation has been updated if needed~
- ~PR title / commit messages contain Jira ticket number if applicable~
- ~Tests have been updated / created if applicable~
- ~Admin Panel has been updated if models have been added or modified~
- ~Migrations have been updated and committed if applicable~
- ~Team Postman workspace has been updated if applicable~

### Reviewers Should Do The Following:
- [x] Review the code changes here
- [ ] Pull the branch and test locally

### Additional Notes
N/A

### CODE REVIEW

File: firstvoices/backend/views/search/base_search_views.py [hunk 1]
```diff
@@ -256,6 +256,78 @@
...
+                    OpenApiExample(
+                        "False",
+                        value=False,
+                        description="No hasVideo filter applied.",
+                    ),
+                    OpenApiExample(
+                        "Oranges",
+                        value=False,
+                        description="Invalid input, defaults to false.",
+                    ),
```
Comment (sarahfirstvoices) [inline firstvoices/backend/views/search/base_search_views.py:328]: Did we already switch all the boolean filters to allow true/false/unset? If so I think these docs need to make that clear-- I would expect the default and the "Oranges" value to be null, not false

Comment (NolanVH) [reply firstvoices/backend/views/search/base_search_views.py:328]: We haven't added that functionality yet. Currently we only allow true or false. The ticket to add unset is [FW-5183](https://firstvoices.atlassian.net/browse/FW-5183).

**Review by bronson-c — APPROVED** [review]
LGTM!

**Review by sahil97 — APPROVED** [review]
LGTM !

**Pull request merged**